README.md

```diff
@@ -1 +1,152 @@
-# CrystFEL-Jupyter-utilities
\ No newline at end of file
+# Description:
+
+
+H5see is a project with graphical tools from CrystFEL implemented in Python.
+The project is intended to work as a wrapper for CrystFEL tools when it is not 
+possible to use a graphical environment but graphical (interactive) features 
+are needed. The tools were written to resemble original CrystFEL tools while 
+at the same time feature some of the requests from CrystFEL users. 
+In particular following tools are available:
+hdfsee
+check-peak-detection.sh (script)
+cell_explorer
+
+Is is recommended to cite the original CrystFEL papers if one is planning to 
+refer to the h5see package:
+
+* T. A. White, V. Mariani, W. Brehm, O. Yefanov, A. Barty, K. R. Beyerlein, F. Chervinskii, L. Galli, C. Gati, T. Nakane, A. Tolstikova, K. Yamashita, C. H. Yoon, K. Diederichs and H. N. Chapman. "Recent developments in CrystFEL". J. Applied Crystallography 49 (2016) p680-689.
+* O. Yefanov, V. Mariani, C. Gati, T. A. White, H. N. Chapman and A. Barty. "Accurate determination of segmented X-ray detector geometry". Optics Express 23 (2015) p28459-28470.
+* T. A. White. "Post-refinement method for snapshot serial crystallography". Phil. Trans. Roy. Soc. B 369 (2014) 20130330.
+* T. A. White, A. Barty, F. Stellato, J. M. Holton, R. A. Kirian, N. A. Zatsepin and H. N. Chapman. "Crystallographic data processing for free-electron laser sources". Acta Cryst. D69 (2013), p1231–1240.
+* T. A. White, R. A. Kirian, A. V. Martin, A. Aquila, K. Nass, A. Barty and H. N. Chapman. "CrystFEL: a software suite for snapshot serial crystallography". J. Appl. Cryst. 45 (2012), p335–341.
+* R. A. Kirian, X. Wang, U. Weierstall, K. E. Schmidt, J. C. H. Spence, M. Hunter, P. Fromme, T. White, H. N. Chapman and J. Holton. "Femtosecond protein nanocrystallography - data analysis methods". Optics Express 18 (2010) p5713. 
+
+
+
+
+## Tools: 
+
+* hdfsee.py - for displaying a diffraction image from a h5 file.
+
+* GUI_tools.py - displaying histograms from a indexing.stream file (produced by indexamajig) and fitting gaussian curves.
+  
+
+  
+
+## Dependencies:
```

Comment:
Add a setup.py or requirements.txt file with these dependencies, what I found from trying to run the tests:

* matplotlib
* numpy
* jupyter
* h5py
* scipy

Maybe a makefile for CrystFEL to be install into the repo directory, with the folder put in gitignore?

Reply:
A setup.py is already prepared but we wanted to include it with a new branch and a separate MR. Do you think it should be added now?

As to the CrystFEL makefile; a makefile would make sense for 0.7.0 but in 0.8.0 the installation is done with cmake so I think a link  to the installation manual is better (especially if the installation changes in future).